inkycal/main.py

```diff
@@ -104,7 +104,7 @@ def __init__(self, settings_path:str or None=None, render:bool=True):
             os.mkdir(image_folder)
 
         # Option to use epaper image optimisation, reduces colours
-        self.optimize = True
+        self.optimize = False
```

Comment:
The optimisation is only for non-parallel displays. This should be controlled by a flag rather than fully disabling it as it renders text pretty badly on all spi displays. Parallel displays do not face this issue as they support 16 grayscales, which renders font much better by default

Reply:
Thank you. Yes, please revert this. I'll add this feature later on as I'll either have to add this to the driver file or make a optional argument specifying if optimisation should be activated